packages/frontend/src/components/LoginForm.tsx

```diff
@@ -339,17 +359,19 @@ export default function LoginForm({ credentials, setCredentials }: LoginProps) {
 }
 
 interface ConfigureProgressDialogProps {
-  credentials?: Partial<Credentials>
+  credentials?: Credentials
   onSuccess?: () => void
   onUserCancellation?: () => void
   onFail: (error: string) => void
+  proxyUpdated: boolean
 }
 
 export function ConfigureProgressDialog({
-  credentials = {},
+  credentials = defaultCredentials(),
```

Comment:
Wouldn't a nullable type make more sense?

Reply:
It is nullable, but we set a default value here. Let's dicuss that in the follow up